src/common/rbac.ts

```diff
@@ -11,51 +11,190 @@ export type KubeResource =
-  "horizontalpodautoscalers": { kind: "HorizontalPodAutoscaler", group: "autoscaling" },
-  "ingresses": { kind: "Ingress", group: "networking.k8s.io" },
-  "jobs": { kind: "Job", group: "batch" },
-  "namespaces": { kind: "Namespace" },
-  "limitranges": { kind: "LimitRange" },
-  "leases": { kind: "Lease" },
-  "networkpolicies": { kind: "NetworkPolicy", group: "networking.k8s.io" },
-  "nodes": { kind: "Node" },
-  "persistentvolumes": { kind: "PersistentVolume" },
-  "persistentvolumeclaims": { kind: "PersistentVolumeClaim" },
-  "pods": { kind: "Pod" },
-  "poddisruptionbudgets": { kind: "PodDisruptionBudget", group: "policy" },
-  "podsecuritypolicies": { kind: "PodSecurityPolicy", group: "policy" },
-  "priorityclasses": { kind: "PriorityClass", group: "scheduling.k8s.io" },
-  "runtimeclasses": { kind: "RuntimeClass", group: "node.k8s.io" },
-  "resourcequotas": { kind: "ResourceQuota" },
-  "replicasets": { kind: "ReplicaSet", group: "apps" },
-  "roles": { kind: "Role", group: "rbac.authorization.k8s.io" },
-  "rolebindings": { kind: "RoleBinding", group: "rbac.authorization.k8s.io" },
-  "secrets": { kind: "Secret" },
-  "serviceaccounts": { kind: "ServiceAccount" },
-  "services": { kind: "Service" },
-  "statefulsets": { kind: "StatefulSet", group: "apps" },
-  "storageclasses": { kind: "StorageClass", group: "storage.k8s.io" },
+  clusterroles: {
+    kind: "ClusterRole",
+    group: "rbac.authorization.k8s.io",
+    namespaced: false,
+  },
+  clusterrolebindings: {
+    kind: "ClusterRoleBinding",
+    group: "rbac.authorization.k8s.io",
+    namespaced: false,
+  },
+  configmaps: {
+    kind: "ConfigMap",
+    group: "v1",
+    namespaced: true,
+  },
+  cronjobs: {
+    kind: "CronJob",
+    group: "batch",
+    namespaced: true,
+  },
+  customresourcedefinitions: {
+    kind: "CustomResourceDefinition",
+    group: "apiextensions.k8s.io",
+    namespaced: false,
+  },
+  daemonsets: {
+    kind: "DaemonSet",
+    group: "apps",
+    namespaced: true,
+  },
+  deployments: {
+    kind: "Deployment",
+    group: "apps",
+    namespaced: true,
+  },
+  endpoints: {
+    kind: "Endpoint",
+    group: "v1",
+    namespaced: true,
+  },
+  events: {
+    kind: "Event",
+    group: "v1",
+    namespaced: true,
+  },
+  horizontalpodautoscalers: {
+    kind: "HorizontalPodAutoscaler",
+    group: "autoscaling",
+    namespaced: true,
+  },
+  ingresses: {
+    kind: "Ingress",
+    group: "networking.k8s.io",
+    namespaced: true,
+  },
+  jobs: {
+    kind: "Job",
+    group: "batch",
+    namespaced: true,
+  },
+  namespaces: {
+    kind: "Namespace",
+    group: "v1",
+    namespaced: false,
+  },
+  limitranges: {
+    kind: "LimitRange",
+    group: "v1",
+    namespaced: true,
+  },
+  leases: {
+    kind: "Lease",
+    group: "v1",
+    namespaced: true,
+  },
+  networkpolicies: {
+    kind: "NetworkPolicy",
+    group: "networking.k8s.io",
+    namespaced: true,
+  },
+  nodes: {
+    kind: "Node",
+    group: "v1",
```

Comment:
@Nokel81 Should the group be an empty string? It seems only v1 stuff is not showing up for me.

Reply:
Actually this probably isn't the place that you need to fix but instead the uses of `shouldShowResourceInjectionToken`

Reply:
But yes they probably should be empty here too